docs/getting_started.md

```diff
@@ -4,7 +4,7 @@
 
 ## Prerequisites
 
-1. Python version >= 3.9.18 < 3.12
+1. Python version >= 3.10.18 < 3.12
```

Comment:
Is it necessary to define also the z-stream version? If yes, I don't think `.18` is the correct, when latest one is `3.10.17` :smiley: 
```suggestion
1. Python version >= 3.10 < 3.12
```